appserver/common/container-common/pom.xml

```diff
@@ -146,6 +146,18 @@
             <artifactId>ha-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>fish.payara.payara-modules</groupId>
+            <artifactId>hazelcast-bootstrap</artifactId>
+            <version>${project.version}</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
```

Comment:
I thought the decision had been made not to use lombok in Payara

Reply:
News to me... it would add 100s of lines of code, especially for every @Delegate

Reply:
useless code at that...